README.md

```diff
@@ -1,205 +1,278 @@
+
+Here are some guidelines to assist in R code development:
+
+* If your code is a feature, open up a new project on GitHub and sketch out a plan (specific functions) for how you will implement new functionality. Ask for review from your supervisor.
+* Take advantage of [RStudio's debugger](https://support.rstudio.com/hc/en-us/articles/205612627-Debugging-with-RStudio) to walk through functions with similar functionality.
+  - Note: Ironically, sometimes the debugger can be buggy. With patience, you can still walk through functions.
+* Test your new code within an integration project.  
+  - Copy the test project from within the _/Git/_ folder to your _/Desktop_ so that changes to input files aren't integrated onto Git.
+  - Ensure you can run the test project in RStudio prior to any code development. See User Manual (todo) for running a test project.
+  - Change or set options in the descriptions.R file so that your new functionality is triggered (pertains to rSFSW2).
+  - Switching on and off of certain options should NOT be added into _GIT_. If a _new_ option is added though, this should be permanently integrated into the code base (i.e. for rSFSW2 in the _/demo_ folder and the _test project_ folder).
+* Develop code in Atom. To test code, re-install the R package and then re-run the test project within RStudio.
+* Document your code both formally (see [R documentation](#rdoc)) and informally (in-line with code explaining reasoning using #).
+* Run the test project again with your new functionality. Because of the labyrinthian nature of the code base you may have missed something you need to change and will receive an error message. Decipher and fix error message, re-install, and try again. shift + Ctrl + F is your friend.
+* Ensure your new code adheres to our style guide. Take advantage of lintr.
+* Ensure your package passes all pre-existing unit tests.
+  - Note: Never turn off a unit test and push this to GitHub. If a unit test is failing seek to understand whether your code is unintentionally changing outcomes (__likely__) or whether a unit test needs updating (this is __unlikely__ but would occur if you explicitly changed the function _with_ the now failing unit test).
+* Write your own unit tests for your new functionality.
+* Clean up code before committing (i.e. deleting print statements, double check with lintr, mistaken .Rproj files).
+* Once your code is functional, refer to the [github_workflow](#github_workflow.md) for next steps on committing and continuous integration checks.
+
+#### Documentation <a name="rdoc"/>
+
+We use the Roxygen2 package to write documentation in R. A manual and examples can be found [here](http://kbroman.org/pkg_primer/pages/docs.html).
```

Comment:
The canonical link is https://CRAN.R-project.org/package=roxygen2
I find the formatting cheat-sheet rather useful: https://cran.r-project.org/web/packages/roxygen2/vignettes/formatting.html